.github/workflows/gh_pages.yml

```diff
@@ -34,6 +30,7 @@ jobs:
         chmod 600 ~/.ssh/id_rsa
         eval "$(ssh-agent -s)"
         ssh-add -k ~/.ssh/id_rsa
+        git clone git@github.com:intel/llvm-docs.git docs
         cd $GITHUB_WORKSPACE/docs
         yes | \cp -rf $GITHUB_WORKSPACE/build/tools/sycl/doc/doxygen/html/* .
```

Comment:
Curious... Do you have an alias for `cp` and this is why you want the `\`?

Reply:
@keryell some distros do. Since I can not control the environment on those machines, I prefer to use `\cp`.